vllm/lora/models.py

```diff
@@ -428,6 +448,7 @@ def __init__(
         # Dict instead of a Set for compatibility with LRUCache.
         self._active_loras: Dict[int, None] = {}
         self._last_mapping: Optional[LoRAMapping] = None
+        self._convert_flag = True
```

Comment:
what is this for? can you add a comment?

Reply:
dirty code, deleted already